src/types.ts

```diff
@@ -67,11 +67,11 @@ export type UseResizableProps = {
   /**
    * callback when border position changes start
    */
-  onResizeStart?: () => void;
+  onResizeStart?: (position: number) => void;
```

Comment:
@flappyBug 
Thank you for the intriguing feature❤️ 

One suggestion: considering the potential addition of parameters in the future, how about making the arguments into an object?

```ts
  export type ResizeCallbackArgs = {
    position: number;
  }
  // ...
  onResizeStart?: (args: ResizeCallbackArgs) => void;
  onResizeEnd?: (args: ResizeCallbackArgs) => void;
```

Reply:
Sure. Thanks for the suggestion. Updated